elide-contrib/elide-test-helpers/src/main/java/com/yahoo/elide/contrib/testhelpers/graphql/elements/VariableDefinitions.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.testhelpers.graphql.elements;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * {@link VariableDefinitions} is a serializable object that models the concept of GraphQL {@code variableDefinitions
+ * } in its grammar.
+ * <p>
+ * According to GraphQL grammar (6.0)
+ * <pre>
+ * {@code
+ * variableDefinitions : '(' variableDefinition+ ')';
+ * }
+ * </pre>
+ * {@link VariableDefinitions} is a space-separated list of {@link VariableDefinition}s surrounded by a pair of
+ * parenthesis.
+ */
+@RequiredArgsConstructor
+public class VariableDefinitions implements Serializable {
```

Comment:
Why not have this import from Definition.  The toResponse method can be defaulted in Definition to throw an UnsupportedOperationException